extensions/amp-social-share/0.1/amp-social-share.css

```diff
@@ -70,3 +70,7 @@ amp-social-share {
 .amp-social-share-whatsapp {
   background-color: #25d366;
 }
+
+.amp-social-share-system {
```

Comment:
It's in [third_party/optimized-svg-icons/amp-social-share-svgs.css](https://github.com/ampproject/amphtml/pull/6688/files#diff-a6dd12bee6d97a0ead117259d76044d4) with the rest of the SVGs—is that what you mean? (Converted from https://material.io/icons/.)